applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _UNICAST_SERVER_H_
+#define _UNICAST_SERVER_H_
+
+#include "le_audio.h"
+
+#include <audio_defines.h>
+
+/**
+ * @brief	Get configuration for audio stream.
+ *
+ * @param[out]	bitrate		Pointer to the bitrate used; can be NULL.
+ * @param[out]	sampling_rate_hz	Pointer to the sampling rate used; can be NULL.
+ * @param[out]	pres_delay_us	Pointer to the presentation delay used; can be NULL.
+ *
+ * @retval	0		Operation successful.
+ * @retval	-ENXIO		The feature is disabled.
+ * @retval	-ENOTSUP	The feature is not supported,
+ * @retval	error		Otherwise
+ */
+int unicast_server_config_get(uint32_t *bitrate, uint32_t *sampling_rate_hz,
+			      uint32_t *pres_delay_us);
+
+/**
+ * @brief	Get advertising data from this module.
+ *
+ * @brief       This partial data is used to build a complete extended advertising packet.
+ *
+ * @param [in/out] uuid_buf Buffer being populated with UUIDs.
+ * @param [in/out] adv_buf Buffer being populated with ext adv elements.
+ * @param [in] adv_buf_vacant Number of vacant elements in adv_buf.
+ *
+ *
+ * @return	0 for success, error otherwise.
+ */
+int unicast_server_adv_get(struct net_buf_simple *uuid_buf, struct bt_data *adv_buf,
+			   uint8_t adv_buf_vacant);
+
+/**
+ * @brief	Send data from the LE Audio unicast (CIS) server, if configured as a source.
```

Comment:
```suggestion
 * @brief	Send data from the LE Audio unicast server, if configured as a source.
```
For consistency, relevant several places in this file

Reply:
Discuss

Reply:
I will change.

Reply:
"(CIS)" is not removed

Reply:
As discussed, I have added (BIS) elsewhere.